test/hotspot/jtreg/compiler/c2/MinValueStrideCountedLoop.java

```diff
@@ -31,17 +31,41 @@
  *                   -XX:CompileCommand=compileonly,*MinValueStrideCountedLoop::test*
  *                   compiler.c2.MinValueStrideCountedLoop
  */
+
+/*
+ * @test
+ * @bug 8316719
+ * @summary Loop increment should not be transformed into unsigned comparison
+ *
+ * @run main/othervm -Xcomp -XX:-TieredCompilation -XX:-UseLoopPredicate
```

Comment:
Since `UseLoopPredicate` is C2 specific, you should add a `@requires vm.compiler2.enabled`.